Automattic/wp-calypso
# Step 1 - Remove Block Patterns Registration from ETK - Remove require_once code

Related to https://github.com/Automattic/wp-calypso/pull/84256

## Proposed Changes

* Remove `require_once` code leaving the files

This PR is step 1, and https://github.com/Automattic/wp-calypso/pull/84256 is step 2.

When testing https://github.com/Automattic/wp-calypso/pull/84256 in my sandbox (running `install-plugin.sh editing-toolkit update/remove-block-patterns`), I got the following message:

```
** WARNING **
It is not safe to deploy the removal of a file (block-patterns/class-block-patterns-from-api.php) along with the removal of the require_once() for this file.
Please undo the file removal and deploying it separately. Alternatively, you can create a new diff to remove the require_once() function and deploy that first.

                        require_once __DIR__ . '/block-patterns/class-block-patterns-from-api.php';
``` 

**Note: This PR can only be merged after https://github.com/Automattic/jetpack/pull/34162 is released.**

## Testing Instructions


* Sandbox the public API
* Apply the code in your sandbox by running `install-plugin.sh editing-toolkit update/remove-block-patterns-require`
* Verify that when you access a simple site you don't see the pattern category "Intro" in the editor pattern inserter

## Pre-merge Checklist
